injected/integration-test/test-pages/broker-protection/actions/expectation-actions-conditional-subaction.json

```diff
@@ -0,0 +1,43 @@
+{
+    "state": {
+      "action": {
+        "actionType": "expectation",
+        "id": "2",
+        "expectations": [
+          {
+            "type": "text",
+            "selector": "body",
+            "expect": "How old is John Doe?"
+          }
+        ],
+        "actions": [
+            {
+                "actionType": "click",
+                "id": "1",
+                "choices": [
+                    { 
+                        "condition": {
+                            "left": "40",
```

Comment:
This is the test that ties everything together, but using `${age}` here throws an error ("age not found") not sure if that's due to the way the test harness works, or if there's a genuine issue here.